src/common/shutdown_state_helper.c

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: BSD-3-Clause
+/* Copyright 2025, Hewlett Packard Enterprise Development LP */
+
+/*
+ * shutdown_state_helper.c -- implementation of shutdown_state_reinit
+ */
+
+#include <string.h>
+#include <stdbool.h>
+#include <endian.h>
+#include "shutdown_state.h"
+#include "out.h"
+#include "util.h"
+#include "os_deep.h"
+#include "set.h"
+#include "libpmem2.h"
+#include "bad_blocks.h"
+#include "../libpmem2/pmem2_utils.h"
```

Comment:
```suggestion
#include "out.h"
#include "os_deep.h"
#include "set.h"
```

I believe there is a few too many here.

Reply:
Done.